src/test/activation/migrateDataScienceSettingsService.unit.test.ts

```diff
@@ -15,130 +15,157 @@ suite('Migrate data science settings', () => {
+    "python.experiments.optOutFrom": [
+        "DeprecatePythonPath - experiment"
+    ],
+}`;
+
+    const expectedMigratedSettings = `{
+    "python.languageServer": "Pylance",
+    "python.linting.enabled": false,
+    "python.experiments.optOutFrom": [
+        "DeprecatePythonPath - experiment"
+    ],
+    "jupyter.allowImportFromNotebook": true,
+    "jupyter.alwaysTrustNotebooks": true,
+    "jupyter.enabled": true,
+    "jupyter.jupyterInterruptTimeout": 0,
+    "jupyter.jupyterLaunchTimeout": 0,
+    "jupyter.jupyterLaunchRetries": 0,
+    "jupyter.jupyterServerType": "remote",
+    "jupyter.notebookFileRoot": "foo",
+    "jupyter.changeDirOnImportExport": true,
+    "jupyter.useDefaultConfigForJupyter": true,
+    "jupyter.searchForJupyter": true,
+    "jupyter.allowInput": true,
+    "jupyter.showCellInputCode": true,
+    "jupyter.collapseCellInputCodeByDefault": true,
+    "jupyter.maxOutputSize": 0,
+    "jupyter.enableScrollingForCellOutputs": true,
+    "jupyter.sendSelectionToInteractiveWindow": true,
+    "jupyter.markdownRegularExpression": "foo",
+    "jupyter.codeRegularExpression": "foo",
+    "jupyter.allowLiveShare": true,
+    "jupyter.errorBackgroundColor": "foo",
+    "jupyter.ignoreVscodeTheme": true,
+    "jupyter.variableExplorerExclude": "foo",
+    "jupyter.liveShareConnectionTimeout": 0,
+    "jupyter.decorateCells": true,
+    "jupyter.enableCellCodeLens": true,
+    "jupyter.askForLargeDataFrames": true,
+    "jupyter.enableAutoMoveToNextCell": true,
+    "jupyter.allowUnauthorizedRemoteConnection": true,
+    "jupyter.askForKernelRestart": true,
+    "jupyter.enablePlotViewer": true,
+    "jupyter.codeLenses": "jupyter.runcell, jupyter.runallcellsabove, jupyter.debugcell",
+    "jupyter.debugCodeLenses": "jupyter.debugcontinue, jupyter.debugstop, jupyter.debugstepover",
+    "jupyter.debugpyDistPath": "foo",
+    "jupyter.stopOnFirstLineWhileDebugging": true,
+    "jupyter.textOutputLimit": 0,
+    "jupyter.magicCommandsAsComments": true,
+    "jupyter.stopOnError": true,
+    "jupyter.remoteDebuggerPort": 0,
+    "jupyter.colorizeInputBox": true,
+    "jupyter.addGotoCodeLenses": true,
+    "jupyter.useNotebookEditor": true,
+    "jupyter.runMagicCommands": "foo",
+    "jupyter.runStartupCommands": [
+        "foo",
+        "bar"
+    ],
+    "jupyter.debugJustMyCode": true,
+    "jupyter.defaultCellMarker": "foo",
+    "jupyter.verboseLogging": true,
+    "jupyter.themeMatplotlibPlots": true,
+    "jupyter.useWebViewServer": true,
+    "jupyter.variableQueries": [
+        "foo",
+        "bar"
+    ],
+    "jupyter.disableJupyterAutoStart": true,
+    "jupyter.jupyterCommandLineArguments": [
+        "foo",
+        "bar"
+    ],
+    "jupyter.alwaysScrollOnNewCell": true,
+}`;
+    const KEYBINDINGS_FILEPATH = '/path/to/keybindings.json';
+    const originalKeybindings = `[
+    {
+        "key": "ctrl+shift+enter",
+        "command": "python.dataScience.runallcells"
```

Comment:
Since you're editing settings in this change, perhaps we can fix the code lens commands for values in this change too?

When we ported over the ```python.dataScience.codeLenses```

Reply:
Not sure what this means @rchiodo, would you mind elaborating?

Reply:
Sorry should have linked a bug. That might make it clearer:
https://github.com/microsoft/vscode-jupyter/issues/1088

Okay I'm an idiot. You're fixing that bug.